app/src/full/java/io/homeassistant/companion/android/vehicle/MainVehicleScreen.kt

```diff
@@ -38,26 +57,76 @@ class MainVehicleScreen(
             "script" to commonR.string.scripts,
             "switch" to commonR.string.switches,
         )
+
+        private val DOMAIN_TO_ICON: Map<String, IIcon> = mapOf(
```

Comment:
On Wear this is handled by [creating a placeholder entity](https://github.com/home-assistant/android/blob/220b73d20f53cd73b1fea9e27f62ec638a73516c/wear/src/main/java/io/homeassistant/companion/android/util/CommonFunctions.kt#L33), so we can have all the icon code in one place and match the frontend. Did you consider using this?

Reply:
Done